src/Routing/Router.php

```diff
@@ -926,6 +911,27 @@ public static function parseNamedParams(ServerRequest $request, array $options =
     }
 
     /**
+     * Create a RouteBuilder for the provided path.
+     *
+     * @param string $path The path to set the builder to.
+     * @param array $options The options for the builder
+     * @return \Cake\Routing\RouteBuilder
+     */
+    public static function getRouteBuilder($path, array $options = [])
```

Comment:
Can we rename this to `createRouteBuilder` or `newRouteBuilder`, so that it does not give the impression it is the same one every time?